testing/mock/beacon_chain_service_mock.go

```diff
@@ -13,401 +13,401 @@ import (
 	metadata "google.golang.org/grpc/metadata"
 )
 
-// MockBeaconChain_StreamChainHeadServer is a mock of BeaconChain_StreamChainHeadServer interface
+// MockBeaconChain_StreamChainHeadServer is a mock of BeaconChain_StreamChainHeadServer interface.
```

Comment:
I'm guessing there was an upstream change to mockgen that added all this punctuation when you regenerated?

Reply:
Yep!